dashboard/src/setup.py

```diff
@@ -17,6 +17,7 @@
         'celery==5.2.7',
         'flower==1.2.0',
         'python3-saml==1.15.0',
-        'flask-login==0.6.2'
+        'flask-login==0.6.2',
+        'prometheus-flask-exporter'
```

Comment:
Could you add the version number? Just for consistency's sake

Reply:
good catch! meant to do so after testing it out